src/details/ListingSearch.js

```diff
@@ -240,22 +187,24 @@ const ListingsSearch = () => {
                                             className="fa fa-trash"></i> Search</span>
                                     </div>
                                 </div>
-                                <MultiSelectFilter className='location'/>
-                                <SingleSelectFilter className='keyword'/>
-                                <SingleSelectFilter className='type'/>
-                                <SingleSelectFilter className='make'/>
-                                <SingleSelectFilter className='model'/>
-                                <SingleSelectFilter className='trim'/>
+                                <SingleSelectFilter className='keyword' keyWord={"keyword-collapse-1"} name={"Location"}/>
+                                <SingleSelectFilter className='keyword' keyWord={"keyword-collapse-2"} name={"Dam"}/>
+                                <SingleSelectFilter className='type' keyWord={"keyword-collapse-3"} name={"Sire"}/>
+                                <SingleSelectFilter className='make' keyWord={"keyword-collapse-4"} name={"Age"}/>
+                                <SingleSelectFilter className='model' keyWord={"keyword-collapse-5"} name={"Sex"}/>
+                                <SingleSelectFilter className='trim' keyWord={"keyword-collapse-6"} name={"Price"}/>
+                                <SingleSelectFilter className='model' keyWord={"keyword-collapse-7"} name={"Discipline"}/>
+                                <SingleSelectFilter className='trim' keyWord={"keyword-collapse-8"} name={"Breading Capable"}/>
                             </div>
                         </div>
-                        <div className="col-md-9" style={{margin: 0, padding: 0}}>
+                        <div className="col-md-9 horse-section">
                             <div className="p-3">
-                                <div className="row p-3" style={{backgroundColor: "#464F58", color: "white"}}>
+                                <div className="row p-3 matches">
                                     <div className="row mx-auto">
                                         Showing 1 out of 1 Matches
                                     </div>
                                 </div>
-                                <div className="row p-3" style={{backgroundColor: "#6D7985", color: "white"}}>
+                                <div className="row p-3 sort">
```

Comment:
This is not the line of code this comment is about.

[These](https://github.com/B2Gdevs/horse-market-react/pull/16/files#diff-aa8dcb1c762272acc52c24757baa194ad63959098a8285436845da489d2d9924R211-R243) are the lines of code this comment is about.

The sorting options don't sort anything as of right now.  When the `listingData` object is made, these sorts should be sorting that data and displaying it in a sorted way.